translator/trace/zipkin/status_code.go

```diff
@@ -62,6 +57,21 @@ func (m *statusMapper) ocStatus() *tracepb.Status {
 		s = m.fromHTTP
 	}
 
+	// If no codePtr was provided, fallback to the first source with a message
```

Comment:
applying this default caused this test to fail https://github.com/open-telemetry/opentelemetry-collector/blob/665d64efef4637b8216108ed07d8b93a443833dd/translator/trace/zipkin/zipkinv1_thrift_to_protospan_test.go#L166-L175


```
attributes: {
  "status.message": "Forbidden",
}

expects tracepb.Status  to be nil
```


The jaeger behavior matches the previous zipkin behavior https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/trace/jaeger/jaegerthrift_to_protospan.go#L262-L265 


Do you think I should update the tests or modify the behavior?

Reply:
Please change behavior . We should not set any error status based on "status.message" only

Reply:
Does this only apply to that specific open census attribute? I ask because the existing code will return a tracepb.Status when either a code or message is set `if s.codePtr != nil || s.message != "" `. Because of the way it is currently implemented, this would only ever apply to fromHTTP though. 

On master, the current behavior is 

```
given

attributes: {
   "http.status_message": "something"
}

returns

tracepb.Status{
    Code: 0,
    Message: "something",
}


```

Reply:
You are right. we should not set error status code based only on an error message attribute such as `http.status_message` or `status.message` (hopefully I don't miss any other message attribute)

Reply:
@chris-smith-zocdoc could you apply this approach ^ and make sure tests pass?